Changes from Elizabeth (#3257)
mendix/docs: content/data-hub/data-hub-catalog/search.md

## 1 Introduction

> The search functionality in the [Data Hub Catalog](#data-hub-home) makes finding the right data to use easy. 

Reviewer: ```suggestion
The search functionality in the [Data Hub Catalog](#data-hub-home) makes finding the right data to use in your app development easy. 
```

> The search functionality in the [Data Hub Catalog](#data-hub-home) makes it easy to find the right data to use in your app development. 

Author: I think that's what the table of contents, search, and scrolling through the page are for. Saying something only in one place instead of alluding to it everywhere makes it easier to find the thing you need.

- The details of registered data assets can be viewed in the [Asset details](#search-details) screen. 
- The [Copy Data Source URI](#service-details) and **Download** contract buttons enable you to access the data source endpoints which you can use to integrate registered data sources into your enterprise applications.

> You can start searching from the [Data Hub Home](#data-hub-home) page, or click the [Catalog](#search-tab) tab to go to the **Search** pane and **Asset Details** screen.

Reviewer: ```suggestion
You can start searching from the [Data Hub Home](#data-hub-home) page or click the [Catalog](#search-tab) tab to go to the **Search** pane and **Asset Details** screen.
```

{{% alert type="info" %}}The [Data Hub pane](/refguide/data-hub-pane) in Studio Pro integrates the powerful search of the Data Hub Catalog. You can find and use registered datasets as external entities in your app development.{{% /alert %}}

> Registered assets can be [curated](#curate-bar) to add and edit **Tags**, **Owners**, and **Descriptions**. You can also set properties to  **Discoverable** and **Validated** to ensure the right people can find them.

Reviewer: ```suggestion
Registered assets can be [curated](#curate-bar) to add and edit **Tags**, **Owners**, and **Descriptions**. Further curate functions include setting properties such as  **Discoverable** and **Validated** . 
```

## 2 Details of Registered Assets

> The Catalog displays the [details](#search-details) of data sources, datasets, and attributes from the published OData service contract. This section describes important properties of registered contracts.

Reviewer: Here I want to emphasise that they are going to be adding to the Catalog information for the registered source - therefore not changing the metadata contract, or affecting the data.

Author: Oh, that message did not come across to me. What about:

```suggestion
The Catalog displays the [details](#search-details) of data sources, datasets, and attributes as provided in the published OData service contract. You can also add metadata in the Catalog without affecting the contract or the data. This section describes important properties of registered assets.
```

Reviewer: ```suggestion
The Catalog displays the [information](#search-details) about data sources, datasets, and attributes (the metadata). This section describes important properties of registered contracts.
```

> {{% alert type="info" %}}The **Dataset** is the name of the **Entity set** of a published **Entity** in Mendix Studio Pro. By default, the entity name has an "s" appended to it. For example, if an entity named `Customer` is published in an OData service, the **Dataset** name in the **Search Details** will be `Customers`.{{% /alert %}}

Reviewer: ```suggestion
{{% alert type="info" %}}The **Dataset** is the name of the **Entity set** of a published **Entity** in Mendix Studio Pro, which by default, is the entity name with an "s" appended to it. For example, if an entity named `Customer` is published in an OData service, the **Dataset** name in the **Search Details** will be `Customers`.{{% /alert %}}
```

### 2.1 Versions 

> Every published OData service/data source has a version number. Apps that consume a data source will consume from a specific version. Updates and changes to a service will be indicated by a change in the version number (if the data source originators follow good practices). This may result in several versions of a registered data source available in the Catalog. Each version be listed as a separate item in the search results.

Reviewer: ```suggestion
Every published OData service (which is referred to as a data source in Data Hub) has a version number and apps that consume a data source will consume a specific version. When the data source owner issues updates and changes to a service, if good practice is followed, this will be indicated by a change in the version number. If the changed Odata services is deployed to a different endpoint this will result in several versions that are available in the Catalog. Each endpoint will be listed as a separate item in the search results.
```

> Every published OData service/data source has a version number. Apps that consume a data source will consume from a specific version. Updates and changes to a service will be indicated by a change in the version number (if the data source originators follow good practices). This may result in several versions of a registered data source available in the Catalog. Each version be listed as a separate item in the search results.

Author: Shorter sentences are more accessible. Here's the United States government recommendation on this topic: https://www.plainlanguage.gov/.

### 2.1 Versions 

Every published OData service or data source (as they are known in the Catalog) has a version number, and apps that consume a datasource will consume from a specific version. Updates and changes to a service will be indicated by a change in the version number if good practice is followed by the data source originators. This may result in several versions of a registered data source available in the Catalog that will all be listed as separate items in the search results for the same-named data source.
Every published OData service/data source has a version number. Apps that consume a data source will consume from a specific version. Updates and changes to a service will be indicated by a change in the version number (if the data source originators follow good practices). This may result in several versions of a registered data source available in the Catalog. Each version be listed as a separate item in the search results.

The version of the selected data source is displayed in the [Asset Details](#search-details).

### 2.2 Environments

> The Data Hub Catalog is a register of apps that are deployed to a particular environment and the services or data sources published from the apps that are deployed to the same environment. This means that each registered data source is a unique **endpoint** which is the location of the OData service contract that includes the version of the service running in a specific environment.

Author: Using register as both a noun and a verb is confusing. We have a registration API, which people use to register. I was just searching for some other noun.

> The Data Hub Catalog is a register of apps that are deployed to a particular environment and the services or data sources published from the apps that are deployed to the same environment. This means that each registered data source is a unique **endpoint** which is the location of the OData service contract that includes the version of the service running in a specific environment.

Reviewer: I wanted to indicate and emphasize registration and that it is a Catalog (which was my dilema - not to call it a "catalog of ....)

![](attachments/search/curate-bar.png)

Owners and curators can carry out the following actions:

* **Edit Metadata** – edit information that is displayed in the Catalog for the asset: 
  * for a selected data source you can edit [Application Details](./curate#curate-application) and [Data Source Details](./curate#service-details)
  *  for a selected dataset you can edit [Dataset Details](./curate#curate-datasets)
* **Discoverable**/**Validated** – set the discoverability of the service, and validate the dataset

>   * **Discoverable** – all users of the Data Hub Catalog and Mendix Studio Pro can see and consume the service 

Reviewer: ```suggestion
  * **Discoverable**:  **On** - all users can see the service; **Off** the is not visible to users, only the owners, curators, and the Data Hub Admin will have access to the service 
```

>   * **Not Discoverable** – the service is not visible and only owners of the service, Data Hub curators, and the Data Hub Admin can access the service

Reviewer: ```suggestion
 
```

>   * **Validated** – indicates if the dataset has been validated

Reviewer: ```suggestion
  * **Validated** – when set, indicates if the dataset has been validated
```

> {{% alert type="info" %}}By default, newly registered services are set to **Discoverable** and visible to all users and are set to **Not validated**.{{% /alert %}}

Reviewer: ```suggestion
{{% alert type="info" %}}Newly registered data sources are set to **Discoverable** and the **Validated** is not active.{{% /alert %}}
```
